reana/reana_dev/git.py

```diff
@@ -371,8 +376,40 @@ def git_clone(user, component, exclude_components):  # noqa: D301
     if exclude_components:
         exclude_components = exclude_components.split(",")
     components = select_components(component, exclude_components)
+
+    instalation_directory = os.path.realpath(target_directory+'/')
+
+    def yes_no_question():
+        check = str(input("Directory {0} empty. Do you want to clone REANA components here? [y/n] ".format(instalation_directory))).lower().strip()
+        try:
+            if check[0] == 'y':
+                return True
+            elif check[0] == 'n':
+                return False
+            else:
+                return yes_no_question()
+        except Exception as error:
+            return yes_no_question()
+
+        
+    if len(os.listdir(instalation_directory)) == 0:
+        clone_answer = yes_no_question()
+    else:
+        now = datetime.datetime.now().strftime("%Y-%m-%dT%H:%M:%S")
```

Comment:
Perhaps you can use `display_message()` function from `utils.py` that would do the formatting nicely?  So that it is centralised in one place...

Reply:
Done